oeps/oep-0025-proc-incremental-improvements.rst

```diff
@@ -26,6 +26,23 @@ OEP-25: Incremental Improvements
 
 .. _Original pull request: https://github.com/edx/open-edx-proposals/pull/66#pullrequestreview-151034037
 
+TL;DR
+=====
```

Comment:
Very minor nit: headings read better in the file if you leave a blank line after them.